lib/rust/transformer/src/main.rs

```diff
@@ -253,14 +254,16 @@ async fn read_events_s3<'a>(
     Pin<Box<dyn Stream<Item = Result<Value>> + Send>>,
 )> {
     println!("Starting download");
+    let dec_key = decode(&r.key.clone()).unwrap().to_string();
```

Comment:
Just use ? here, unwrap will panic the program (we need to reduce our usage elsewhere too :)). You can use `.map_err(|e|anyhow!(e))` if the types are incompatible (or `.context("...")?` if ur unwrapping an Option.

Reply:
Oh great catch, I'll clean this up in the am